TemplateProject/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "semi": false,
+  "singleQuote": false,
```

Comment:
I _think_ this is the default? I could be wrong.

Reply:
You are correct!